docs/site/Controller-generator.md

```diff
@@ -216,6 +216,24 @@ export class TodoController {
     return this.todoRepository.findById(id, filter);
   }
 
+  @get('/todos/findOne', {
+    responses: {
+      '200': {
+        description: 'Todo model instance',
+        content: {
+          'application/json': {
+            schema: getModelSchemaRef(Todo, {includeRelations: true}),
+          },
+        },
+      },
+    },
+  })
+  async findOne(
+    @param.where(Todo) where?: Where<Todo>
+  ): Promise<Todo | null> {
+    return this.todoRepository.findOne(where);
```

Comment:
What is the HTTP response when no model was found? Do we return `200 OK` and `null` in the response body? Should we return `404 Not Found` instead?

I feel it's important to have a test for this edge case. Since we don't run HTTP tests for controllers scaffolded by CLI, it may be easier to add `findOne` to [`TodoController`](https://github.com/strongloop/loopback-next/blob/85a616415253bdca0475b7db9072da1ee611ac28/examples/todo/src/controllers/todo.controller.ts)  and then add the test to [examples/todo/src/__tests__/acceptance/todo.acceptance.ts](https://github.com/strongloop/loopback-next/blob/85a616415253bdca0475b7db9072da1ee611ac28/examples/todo/src/__tests__/acceptance/todo.acceptance.ts).